### DISCUSSION TITLE
How to specify linter for Python language?

### DISCUSSION THREAD

**Discussion by Tom-Boscher** [zed-industries/zed#6545] [Feature Requests]
I use Black formatter for Python formatting:
```
  "language_overrides": {
    "Python": {
      // "format_on_save": "off",
      "formatter": {
        "external": {
          "command": "black",
          "arguments": [
            "-"
          ]
        }
      }
    }
  }
```
I want to use Ruff linter for Python language but I don't really understand how to setup linter with Zed. Do I have to specify linter in Pyright settings or in Zed settings?

**Comment by hovsater** [comment]
I believe this [previous discussion](https://github.com/zed-industries/community/discussions/1124) is of interest. 🙂

**Reply by Tom-Boscher** [reply]
Oh thank you!
